models/ncgamesmodel.js

```diff
@@ -17,3 +17,28 @@ exports.fetchReviewByID = (reviewID) => {
       });
   }
 };
+exports.patchReview = (reviewID, newVote) => {
+  let valuesArr = [];
+  if (reviewID) {
+    valuesArr.push(reviewID);
+    return connection
+      .query("SELECT * FROM reviews WHERE review_id = $1", valuesArr)
+      .then((results) => {
```

Comment:
you don't need to be doing an extra `SELECT` here 

you can use data in the database (i.e `votes`) in an `UPDATE`!


> if we were trying to add to a column in a partys table
```sql
UPDATE partys
SET attendees = attendees + 1
WHERE party_id = 3;
```